en/middleware/micrortps.md

```diff
@@ -0,0 +1,344 @@
+# RTPS/ROS2 interface: the PX4-FastRTPS bridge
+
+This bridge adds communication capabilities between a *PX4 Autopilot* and a *Fast RTPS* application, through serial ports or
```

Comment:
I think this whole section needs work because it isn't clear why you would bother creating a DDS service. 

As I understand it the point of RTPS is to be able to publish and subscribe uORB topics beyond the flight controller so that external components can subscribe as though they were on the flight controller. This allows deep integration with simulators and off board APIs. 

Furthermore that we have RTPS that is a "data neutral" mechanism for sharing data. What this bridge does is serialise our data, create a bridge between uorb and RTPS, and open a transport layer for the messages.